### ISSUE TITLE
Can't vote through lisk desktop wallet

### ISSUE THREAD

**Issue by sacOO7** [LiskHQ/lisk-desktop#3140]
### Expected behavior
Should be able to vote through desktop wallet, also having scrolling issues

### Actual behavior
Not able to vote 
Shows error 
```
Access to XMLHttpRequest at 'https://hub32.lisk.io/api/blocks?generatorPublicKey=875394…&limit=1' from origin 'http://localhost:5659' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

### Steps to reproduce
Click confirm for voting

### Which version(s) does this affect? (Environment, OS, etc...)
Latest version of wallet - 1.27.1,  windows 10

**Comment by yarooo85** [comment]
Also scroll not working when more delegates are selected (Win10, screen 1600*900) which makes submit button unavailable.
![scroll](https://user-images.githubusercontent.com/43033855/97788221-af5ced00-1bb7-11eb-9859-34ecad285960.PNG)

**Comment by reyraa** [comment]
@yarooo85 @sacOO7 Thank you for reporting the issues. I'll make sure we resolve them with the next release.

**Comment by reyraa** [comment]
This issue is resolved by 1.27.2, due to releases soon.
